package.json

```diff
@@ -71,6 +71,7 @@
     "css-loader": "^0.28.7",
     "enzyme": "^3.1.0",
     "enzyme-adapter-react-16": "^1.7.1",
+    "file-loader": "4.0.0",
```

Comment:
I have to manually add this because it's a lodaer that we use, but was never added here because of one of the loaders being removed required it as a dependency